presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -463,6 +465,11 @@ private void updateWrittenBytes()
         writtenBytes = current;
     }
 
+    private void updateNumWrittenFiles()
+    {
+        operatorContext.getRuntimeStats().addMetricValue("numWrittenFiles", NONE, pageSink.getNumWrittenFiles());
```

Comment:
can we make "numWrittenFiles" a constant in `RuntimeMetricName.java` and then use it.